web-portal/backend/src/apps/apps.service.ts

```diff
@@ -83,6 +84,7 @@ export class AppsService {
     if (!tenants) return;
     const newApp = await this.prisma.client.app.create({
       data: {
+        id: nanoid(10),
```

Comment:
Is there any implications of 10 here? I'm not familiar with nanoid, so not sure what benefit this has over 9, or what would be gained by going to 11. This is the type of decision that is good to document so future devs know what's going on. I'd recommend making a constant since this is used in several places (2), and the constant can be named and commented appropriately.

Reply:
https://zelark.github.io/nano-id-cc/

10 character is strong enough to not repeat !